src/templates/layouts/vamcSystem/index.tsx

```diff
@@ -0,0 +1,175 @@
+  useEffect(() => {
+    window.sideNav = menu
+  })
+
+  // TODO: It looks like the reason the image is taller than in the original page is
+  // because the image file is actually different. It has a taller intrinsic height.
+  // Here is an example from `/washington-dc-health-care/`:
+  // Old:
+  // https://s3-us-gov-west-1.amazonaws.com/content.www.va.gov/img/styles/7_2_medium_thumbnail/public/2021-08/Washington%20VA%20Medical%20Center.jpg
+  // New:
+  // https://dsva-vagov-staging-cms-files.s3.us-gov-west-1.amazonaws.com/styles/2_1_large/public/2021-08/Washington%20VA%20Medical%20Center.jpg
+  const hasValidImage = image?.links?.['2_1_large']?.href
+
+  return (
+    <div className="va-l-detail-page va-facility-page">
+      <div className="usa-grid usa-grid-full">
+        {/* Nav data fille in by a separate script from `window.sideNav` */}
+        <nav aria-label="secondary" data-widget-type="side-nav" />
+        {/* Main page content */}
+        <div className="usa-width-three-fourths">
+          <article className="usa-content va-l-facility-detail vads-u-padding-bottom--0">
+            {/* <LovellSwitcher
+              currentVariant={lovellVariant}
+              switchPath={lovellSwitchPath}
+            /> */}
+            {title && <h1>{title}</h1>}
+            {hasValidImage && (
+              <div className="duotone darken lighten medium-screen:vads-u-margin-bottom--0p5">
+                <MediaImage {...image} imageStyle="2_1_large" />
+              </div>
+            )}
+            {/* Was going to use TopTasks, but the links are different, and the wrapper uses different classes */}
+            <div className="usa-grid usa-grid-full vads-u-margin-top--0 vads-u-margin-bottom--3"></div>
+            {introText && (
+              <div className="va-introtext">
+                <p className="vads-u-margin-bottom--0">{introText}</p>
+              </div>
+            )}
+            {/* Locations Section */}
+            <section>
+              <h2 className="vads-u-font-size--xl vads-u-margin-top--3 medium-screen:vads-u-margin-top--5 medium-screen:vads-u-margin-bottom--2p5">
+                Locations
+              </h2>
+              {/* {mainFacilities?.entities?.map((facility) => (
+                <FacilityListing
+                  key={facility.entityId}
+                  {...facility}
+                  fieldVaHealthConnectPhone={fieldVaHealthConnectPhone}
+                />
+              ))} */}
+              <va-link
+                active
+                className="vads-u-font-size--md vads-u-display--block vads-u-width--full"
+                href={`${path}/locations`}
+                text="See all locations"
+              />
+            </section>
+            {/* Manage your health online section */}
+            {administration?.id !== 1039 && (
+              <section>
+                <h2>
+                  {administration?.id === 1040
```

Comment:
Oof. Magic numbers. Can we define these as `consts` with meaningful names?

@laflannery or @omahane Do you know offhand what these magic numbers are?

Reply:
Meaningful names would be nice. Good idea

Reply:
Those are the entity IDs for [Lovell TRICARE](https://prod.cms.va.gov/section/vha/vamc-facilities/visn-12/lovell-tricare) and [Lovell VA](https://prod.cms.va.gov/section/vha/vamc-facilities/visn-12/lovell-va). You can see them in the Section Taxonomy if you edit the term. There is also [a comment in the content-build file](https://github.com/department-of-veterans-affairs/content-build/blob/d25325957f43309d349d61274537ecd0458683da/src/site/layouts/health_care_region_page.drupal.liquid#L55) that might be useful to include here to remind us later that this is to hide this section from TRICARE

Reply:
Without looking at the code, I was almost certain these were Lovell. Thanks for confirming, @laflannery .

Reply:
Thank you!

Reply:
https://github.com/department-of-veterans-affairs/next-build/pull/1017/commits/78ac4f4278c4910cee133be9e67e5aa9fdb55d61